src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable } from '@angular/core';
+
+import * as _ from 'lodash';
+import { ToastsManager } from 'ng2-toastr';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
+import { NotificationType } from '../enum/notification-type.enum';
+import { CdNotification } from '../models/cd-notification';
+
+@Injectable()
+export class NotificationService {
+  // Observable sources
+  private notificationDataSource = new BehaviorSubject<CdNotification[]>([]);
```

Comment:
This name should be shorten as we don't need the notification prefix in the notification service.

Reply:
Done